website/docs/reference/deprecations.md

```diff
@@ -58,6 +58,42 @@ Some deprecations can be automatically fixed with a script. Read more about it i
 
 The following are deprecation warnings in dbt today and the associated version number in which they first appear.
 
+### ArgumentsPropertyInGenericTestDeprecation
+
+dbt has deprecated the ability to specify a custom top-level property called `arguments` on generic tests. This deprecation warning is only raised when the behavior flag `require_generic_test_arguments_property` is set to `False`.
+
+
+#### ArgumentsPropertyInGenericTestDeprecation warning resolution
+
+If you previously had a property called `arguments` on custom generic tests:
+
+<File name='model.yml'>
+
+```yaml
+models:
+  - name: my_model_with_generic_test
+    data_tests:
+      - my_custom_generic_test:
+          arguments: [1,2,3]
+          expression: "order_items_subtotal = subtotal"
+```
+
+You should flip the `require_generic_test_arguments_property` flag to `True` and nest any keyword arguments to your test under the new `arguments` property:
+
+<File name='model.yml'>
+
+```yaml
+models:
+  - name: my_model_with_generic_test
+    data_tests:
+      - my_custom_generic_test:
+          arguments: 
+            arguments: [1,2,3]
```

Comment:
@luna-bianca Should there be one `arguments`?

Reply:
updated